test/test_rpc.py

```diff
@@ -214,13 +217,17 @@ def test_py_no_return_result(self):
     @_wrap_with_rpc
     def test_py_function_exception(self):
```

Comment:
I wonder is it possible to get the Python line number about this error too.

You happened to find a good test case. Users passing wrong argument is a common error.

```
def test_py_user_pass_wrong_arguments_exception(self):
```